ClassicAssist/Data/Macros/Commands/ListCommands.cs

```diff
@@ -45,21 +45,23 @@ public static void PushList( string listName, int value )
 
         [CommandsDisplay( Category = nameof( Strings.Lists ),
             Parameters = new[] { nameof( ParameterType.ListName ), nameof( ParameterType.ElementValueFrontBack ) } )]
-        public static int PopList( string listName, string elementValue = "back" )
+        public static int PopList( string listName, object elementValue = null)
         {
+            elementValue = elementValue ?? "back";
+
```

Comment:
Unsure of a better way to solve the default functionality other than null coalescing.